packages/modules/src/modules/epics/ContributionsEpicButtons.tsx

```diff
@@ -1,40 +1,79 @@
 import React, { useEffect } from 'react';
-import { css } from '@emotion/react';
-import { space } from '@guardian/src-foundations';
+import { SerializedStyles, css } from '@emotion/react';
+import { from } from '@guardian/src-foundations/mq';
+import { palette, space } from '@guardian/src-foundations';
 import { Button } from './Button';
 import { EpicVariant, SecondaryCtaType, Tracking, Cta } from '@sdc/shared/types';
 import { addRegionIdAndTrackingParamsToSupportUrl } from '@sdc/shared/lib';
 import { OphanComponentEvent } from '@sdc/shared/types';
 import {
     getReminderViewEvent,
+    OPHAN_COMPONENT_EVENT_APPLEPAY_VIEW,
     OPHAN_COMPONENT_EVENT_CTAS_VIEW,
     OPHAN_COMPONENT_EVENT_REMINDER_OPEN,
 } from './utils/ophan';
 import { useHasBeenSeen } from '../../hooks/useHasBeenSeen';
 import { hasSetReminder } from '../utils/reminders';
 import { isSupportUrl } from '@sdc/shared/dist/lib';
+import { ApplePaySvg } from './ApplePaySvg';
+import { PaymentCardSvg } from './PaymentCardsSvg';
+import { ButtonApplePay } from './ButtonApplePay';
 import { ChoiceCardSelection } from '../shared/helpers/choiceCards';
 
-const buttonWrapperStyles = css`
+const paymentImageStyles = css`
+    display: inline-block;
+    width: auto;
+    height: 25px;
+    margin: ${space[1]}px 0;
+`;
+
+const svgPositionStyles = css`
+    margin: ${space[1]}px -${space[2]}px;
+`;
+
+const buttonWrapperStyles = (showApplePay?: boolean): SerializedStyles => css`
     margin: ${space[6]}px ${space[2]}px ${space[1]}px 0;
     display: flex;
-    flex-wrap: wrap;
-    align-items: center;
+    ${showApplePay
+        ? `flex-direction: column; margin-right: 0px; ${from.desktop} {flex-direction: row;}`
+        : `flex-wrap: wrap; align-items: center;`}
 
     &.hidden {
         display: none;
     }
 `;
 
-const paymentImageStyles = css`
-    display: inline-block;
-    width: auto;
-    height: 25px;
-    margin: ${space[1]}px 0;
+const buttonMarginStyles = (showApplePay?: boolean): SerializedStyles => css`
+    ${showApplePay
+        ? `margin: ${space[1]}px; ${from.desktop} {width: 100%;}`
+        : `margin: ${space[1]}px ${space[2]}px ${space[1]}px 0;`};
 `;
 
-const buttonMargins = css`
-    margin: ${space[1]}px ${space[2]}px ${space[1]}px 0;
+const buttonStyles = (showApplePay?: boolean): SerializedStyles => css`
```

Comment:
Rather than check `showApplePay` here and return null if falsy (`false` or `undefined`), perhaps we should only be calling `buttonStyles` if ``showApplePay` is `true`. If we did that it wouldn't need to be a function, and I'd suggest renaming it to something more meaningful like `ApplePayButtonOverrides`

EG. Convert this...

```
<Button
  onClickAction={url}
  cssOverrides={buttonStyles(true)}
  icon={<PaymentCardSvg cssOverrides={svgPositionStyles} />}
  priority="secondary"
>...</Button>;
```

```
<Button
  onClickAction={url}
  cssOverrides={showApplePay ? ApplePayButtonOverrides : ``}
  icon={<PaymentCardSvg cssOverrides={svgPositionStyles} />}
  priority="secondary"
>...</Button>;
```

Reply:
Re-labeled to a more meaningful `ApplePayButtonOverrides`. css now a const rather then function with an explicit check when called, as shown above.